### ISSUE TITLE
Create Teacher Housing Donation Spot on Donate Page

### ISSUE THREAD

**Issue by ElsieNoel** [alansouzati/zozu-sponsor-portal#6]
We have a spot for donating to building the preschool, but we also need a similar group of elements for our upcoming campaign to build housing for the teachers. That's the next big thing, so to speak. I can send you the copy and photos to use, as well as the appropriate link to the merchant account giving page once you're ready.

**Comment by alansouzati** [comment]
Yeah I'm going to work on this soon. can you send the copy and photos?
